pyrato/parametric.py

```diff
@@ -1,77 +1,10 @@
 # -*- coding: utf-8 -*-
 
-import numpy as np
-from pyrato.rap import reverberation_time_linear_regression
-import warnings
-
-
-def reverberation_time_energy_decay_curve(
-        energy_decay_curve,
-        T='T20'):
-    """Estimate the reverberation time from a given energy decay curve.
-    The linear regression is performed using least squares error minimization
-    according to the ISO standard 3382 [#]_.
-
-    Parameters
-    ----------
-    energy_decay_curve : ndarray, double
-        Energy decay curve. The time needs to be the arrays last dimension.
-    times : ndarray, double
-        Time vector corresponding to each sample of the EDC.
-    T : 'T20', 'T30', 'T40', 'T50', 'T60', 'EDT', 'LDT'
-        Decay interval to be used for the reverberation time extrapolation. EDT
-        corresponds to the early decay time extrapolated from the interval
-        [0, -10] dB, LDT corresponds to the late decay time extrapolated from
-        the interval [-25, -35] dB.
-    normalize : bool, True
-        Normalize the EDC to the steady state energy level
-
-    Returns
-    -------
-    reverberation_time : double
-        The reverberation time
+"""Parametric room acoustics calculations using simple geometric considerations
```

Comment:
then this file can be removed

Reply:
See above, this file will contain things like Sabine's eq. etc.